CMakeLists.txt

```diff
@@ -202,6 +214,8 @@ target_include_directories(${PROJECT_NAME}
         $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/include>
         $<BUILD_INTERFACE:${PROJECT_BINARY_DIR}/include>
         $<INSTALL_INTERFACE:include>
+        $<$<AND:$<BOOL:${BUILD_INTERFACE}>,$<BOOL:${external}>>:${EXTERNAL_TRANSPORT_HEADER_DIRECTORY}>
+        $<$<AND:$<BOOL:${BUILD_INTERFACE}>,$<BOOL:${external}>>:${PROJECT_SOURCE_DIR}/src/c/profile/transport/serial>
```

Comment:
```suggestion
        $<$<AND:$<BOOL:${BUILD_INTERFACE}>,$<BOOL:${_external_transport}>>:${PROJECT_SOURCE_DIR}/src/c/profile/transport/serial>
```

Reply:
In case you need include a header located in _src_ folder, we should move it to _include_ directory.